code/modules/atmospherics/FEA_turf_tile.dm

```diff
@@ -390,6 +393,18 @@ var/global/list/turf/hotly_processed_turfs = list()
 			item.temperature_expose(src.air, src.air.temperature, CELL_VOLUME)
 		temperature_expose(src.air, src.air.temperature, CELL_VOLUME)
 
+	if(src.air.radgas >= RADGAS_MINIMUM_CONTAMINATION_MOLES && !ON_COOLDOWN(src, "radgas_contaminate", RADGAS_CONTAMINATION_COOLDOWN)) //if fallout is in the air, contaminate objects on this tile and consume radgas
+		for(var/atom/movable/item in src)
+			if(isintangible(item) || isobserver(item) || istype(item, /obj/overlay) || istype(item, /obj/effects))
+				continue
```

Comment:
for good measure, probably also exclude anything with `invisibility > INVIS_CLOAK`
Does it make sense in-universe? not really
Would it potentially prevent some "help I'm being irradiated but nothing is there" down the road? Probably

Reply:
can do. Alternate pitch would be making the radioactive glow visible even when the host entity isn't, but I don't think that's actually possible in BYOND

Reply:
Would be possible via `attached_objs`. It's not exactly a clean solution but it is doable.